src/olympia/editors/templates/editors/base.html

```diff
@@ -54,6 +54,10 @@ <h1 class="site-title amo">
           <li><a href="{{ url('editors.queue_auto_approved') }}">
                 {{ _('Auto Approved Add-ons') }} ({{ queue_counts['auto_approved'] }})</a></li>
         {% endif %}
+        {% if action_allowed(amo.permissions.ADDONS_CONTENT_REVIEW) %}
+          <li><a href="{{ url('editors.queue_content_review') }}">
+                {{ _('Content Review') }} ({{ queue_counts['content_review'] }})</a></li>
```

Comment:
I would bring the `</a></li>` to the next line and level it with their opening tags. But damn… that's how we do it everywhere in the file. Nevermind, consistency wins :-/